db.go

```diff
@@ -28,14 +29,16 @@ severity        INTEGER NOT NULL,
 message         TEXT NOT NULL
 );`
 
-const SQL_CREATE_PLUG = `INSERT into plugs (s3id, owner, views, approved)
-VALUES ($1::text, $2::text, $3::integer, false)`
+const SQL_CREATE_PLUG = `INSERT into plugs (s3id, owner, views, approved, shape)
+VALUES ($1::text, $2::text, $3::integer, false, $4::text)`
 
 const SQL_RETRIEVE_APPROVED_PLUGS = `SELECT id, s3id, owner, views FROM plugs WHERE approved=true`
```

Comment:
You need another query to retrieve only "banner" plugs - this is the method currently used by default and it won't filter out non-"banner" plugs. If this query isn't used elsewhere changing it should be fine, but make sure you won't change the behavior of something like the admin page

Reply:
https://github.com/ComputerScienceHouse/csh-plug/pull/8/files#diff-ac94c91f8ea84762d55b6932ef1466a2165984591d3e646360c5bb31b2130900R93-R101